user/docs/faq.md

```diff
@@ -0,0 +1,154 @@
+---
+sidebar_position: 4
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Troubleshooting & FAQ
+
+This page lists some common problems when installing or running NeoForge modpacks. If your problem isn't covered here, please see the [Getting Support][support] section.
+
+## Installing Java or NeoForge
+
+### I downloaded the installer file, but I have no way of running it!
+
+Please see the [Installing Java][installjava] section. Yes, even if you already have Minecraft: Java Edition installed.
+
+### I installed Java, but I'm still getting the wrong version!
+
+This likely means that your PATH variable is not set, or set incorrectly. To do this, do the following depending on your operating system:
+
+<Tabs defaultValue="windows">
+  <TabItem value="windows" label="Windows">
+Download and run the [Jarfix program][jarfix].
+  </TabItem>
+  <TabItem value="macos" label="MacOS">
+Open Finder. In Finder, open the Applications/Utilities folder and double-click Terminal.
+
+Run the following command: `echo export "JAVA_HOME=\$(/usr/libexec/java_home)" >> ~/.bash_profile`
+
+Then, close Terminal and try again.
+  </TabItem>
+  <TabItem value="linux" label="Linux">
+Open the terminal for your Linux distribution. Common names would be `GNOME Terminal` or `Konsole`, however it may vary depending on your exact setup.
+
+Find the location where Java is stored. Often, this will be something like `/usr/lib/jvm/java-21-openjdk-amd64`.
+
+Run the following commands:
+
+- `echo export "JAVA_HOME=/usr/lib/jvm/java-21-openjdk-amd64/" >> ~/.bashrc` (swapping out the path for the actual path if needed)
+- `echo export PATH=$PATH:$JAVA_HOME/bin`
+
+Then, close the terminal and try again.
+  </TabItem>
+</Tabs>
+
+### I am getting an error saying "Could not find or load main class @user_jvm_args.txt", what do I do?
+
+There are two common reasons for this:
+
+- You are running an outdated version of Java. Please see the [Installing Java][installjava] section.
+- You have multiple versions of Java installed on your computer, possibly installed one of them just right now, but your computer is still using the old one. This means that your PATH variable is not set correctly. Please follow the steps outlined [here][wrongjava].
+
+## Playing NeoForge
+
+Sometimes, issues show up not during installation, but during gameplay.
+
+### My game is lagging! What can I do?
```

Comment:
Should we mention the vanilla profiler? I care less about this one since it can only profile up to 10 seconds.

Reply:
I don't really see the point. Telling users to run Spark is what we do already in the support channels.